main/tests/test_views.py

```diff
@@ -18,14 +23,31 @@ def setUp(self):
         self.user = User.objects.create(username='testuser', password='blablabla', is_superuser=True, is_staff=True,
                                         email='testuser@gmail.com', is_active=True)
 
+    @expectedFailure
+    def test_index_view_rendering(self):
+        """Testing main page for render needed posts."""
+        tz = timezone.get_current_timezone()
+        post = Post.objects.create(author=self.user, title='Test', text='superText',
```

Comment:
You can also set `published_date` here, since you're not testing `publish()` method.